Tasks/PublishSymbols/task.json

```diff
@@ -37,25 +37,23 @@
             "label": "Search Pattern",
             "defaultValue": "**\\bin\\**\\*.pdb",
             "required": false,
-            "helpMarkDown": "The pattern used to discover the pdb files to publish.",
-            "groupName": "advanced"
+            "helpMarkDown": "The pattern used to discover the pdb files to publish."
         },
         {
-            "name": "SourceFolder",
-            "type": "filePath",
-            "label": "Path to source folder",
+            "name": "SymbolsFolder",
+            "type": "string",
+            "label": "Path to symbols folder",
             "defaultValue": "",
             "required": false,
-            "helpMarkDown": "The path to the source folder that is searched for symbol files.  The default is $(Build.SourcesDirectory)",
-            "groupName": "advanced"
+            "helpMarkDown": "The path to the folder that is searched for symbol files.  The default is the configured source folder."
         },
         {
-            "name": "SymbolsFolder",
-            "type": "string",
-            "label": "Path to symbols folder",
+            "name": "SourceFolder",
+            "type": "filePath",
+            "label": "Path to source folder",
             "defaultValue": "",
             "required": false,
-            "helpMarkDown": "The path to the folder that is searched for symbol files.  The default is the configured source folder.",
+            "helpMarkDown": "The path to the source control mapping folder. This option should only be used if the source code being indexed is from a different repository. Otherwise defaults to $(Build.SourcesDirectory)",
             "groupName": "advanced"
```

Comment:
any reason we remove this option from advanced group? since it's not required and default to sourceFolder.